ports/stm32/boards/NUCLEO_WB55/rfcore_firmware.py

```diff
@@ -339,8 +414,17 @@ def _write_failure_state(reason):
     return reason
 
 
+def _read_addr():
```

Comment:
Would be good to come up with a better name for this... maybe `{read,write}_target_addr` ?